valhalla/baldr/graphtileheader.h

```diff
@@ -614,7 +630,7 @@ class GraphTileHeader {
   uint64_t nodecount_ : 21;             // Number of nodes
   uint64_t directededgecount_ : 21;     // Number of directed edges
   uint64_t predictedspeeds_count_ : 21; // Number of predictive speed records
-  uint64_t spare1_ : 1;
+  uint64_t has_osmids_ : 1;             // (bool) Whether the tile has OSM ids
```

Comment:
same nitpick, we should be specific that its node ids, since we already have the way ids

Reply:
Agreed.

Reply:
This is no longer required.